test/planFinder.test.js

```diff
@@ -40,14 +40,16 @@ function assertPlan(plan, string, obj) {
 
 // Simplify transformers to have input/output be numbers instead of objects
 describe("PlanFinder simplified transformer tests", function() {
-    it("Sanitizes its inputs", function() {
+    it("Sanitizes its inputs", async function() {
         const planFinder = new PlanFinder({});
-        assert.throws(()=>planFinder.findBestPlan(), SourceCorruptError, "No input or outputInstructions should throw an error");
-        assert.throws(()=>planFinder.findBestPlan({},{}), SourceCorruptError, "No input or outputInstructions types should throw an error");
-        assert.throws(()=>planFinder.findBestPlan({type:0},{}), RenditionFormatUnsupportedError, "No outputInstructions type should throw an error");
-        assert.throws(()=>planFinder.findBestPlan({},{type:0}), SourceCorruptError, "No input type should throw an error");
-        assert.throws(()=>planFinder.findBestPlan({type:0},{type:'not a mime type!'}), RenditionFormatUnsupportedError, "Mime types shouldn't have spaces");
-        assert.throws(()=>planFinder.findBestPlan({type:0},{type:0}), RenditionFormatUnsupportedError, "Types are valid but fails to create plan for other reasons");
+
+        await assert.rejects(async()=>planFinder.findBestPlan(), SourceCorruptError, "No input or outputInstructions should throw an error");
```

Comment:
FYI: You can await `planFinder.findBestPlan()` directly too, like this (as an example):
```suggestion
       assert.rejects(async()=>await planFinder.findBestPlan(), SourceCorruptError, "No input or outputInstructions should throw an error");
```

Reply:
this style was taken directly from the documentation: https://nodejs.org/api/assert.html#assert_assert_rejects_asyncfn_error_message
```
await assert.rejects(
  async () => {
    throw new TypeError('Wrong value');
  },
  {
    name: 'TypeError',
    message: 'Wrong value'
  }
);
```

Reply:
Just an `FYI`, not a change request...